### PR TITLE
SwiftLint Rules

### PR DESCRIPTION
**Pull request by AngelaLee18**
**Project initialization**

**DESCRIPTION**

In this branch we created the project with the correct structure using clean architecture and add swiftlint to the project.

**EVIDENCE**

Screen Shot 2022-10-11 at 11 59 56

Screen Shot 2022-10-11 at 12 00 22

### CODE REVIEW

**Comment by ezequielvelezglobant** [comment]
I should be able to see swiftlint file in the project inspertor like this 
image

File: TheMovieDB/.swiftlint.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+disabled_rules: # rule identifiers to exclude from running
+  - variable_name
+  - nesting
+  - function_parameter_count
```
Comment (ezequielvelezglobant) [inline TheMovieDB/.swiftlint.yml]: why this are disable?

File: TheMovieDB/.swiftlint.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+opt_in_rules: # some rules are only opt-in
+  - control_statement
+  - empty_count
+  - trailing_newline
+  - colon
+  - comma
+included: # paths to include during linting. `--path` is ignored if present.
+  - Project
+  - ProjectTests
+  - ProjectUITests
```
Comment (ezequielvelezglobant) [inline TheMovieDB/.swiftlint.yml]: Wrong references

**Review by ezequielvelezglobant — COMMENTED** [review]
The Scrip should run every time the code compies, not only in first install.
Also there are some missing rules like force_unwraping, implicitly_unwrapped_optional, etc.
And remove commented lines.

File: TheMovieDB/.swiftlint.yml (hunk 1 above)
Comment (AngelaLee18) [reply TheMovieDB/.swiftlint.yml]: fe285245206d487d707ed41abb05ddab4ff4a81a

File: TheMovieDB/.swiftlint.yml (hunk 2 above)
Comment (AngelaLee18) [reply TheMovieDB/.swiftlint.yml]: fe285245206d487d707ed41abb05ddab4ff4a81a

**Review by ezequielvelezglobant — APPROVED** [review]

**Review by agustingodoyc — APPROVED** [review]

**Pull request merged**